Jenkinsfile

```diff
@@ -100,6 +100,7 @@ pipeline {
                         sh 'cd build && gtimeout 1h make check-tpl'
                         sh 'cd build && gtimeout 10m python3 ../script/testing/junit/run_junit.py --build-type=debug --query-mode=simple'
                         sh 'cd build && gtimeout 10m python3 ../script/testing/junit/run_junit.py --build-type=debug --query-mode=extended'
+                        sh 'cd build && gtimeout 10m python3 ../script/testing/oltpbench/run_oltpbench.py --build-type=debug --config-file=../script/testing/oltpbench/configs/test/tpcc.json' 
```

Comment:
Why are we mixing the oltp_bench tests with the JUnit tests?

Reply:
I think Andy mentioned this requirement here.
https://github.com/cmu-db/terrier/issues/1054#issue-671665086 
"we should add additional runs of TPC-C using OLTP-Bench in the Jenkinsfile 'Test' stage with these different modes:"

Reply:
Ah I interpreted that as the End-to-End Debug phase. Maybe I got that wrong